### PR TITLE
Favo 7518 release python sdk v123 iain

### PR DESCRIPTION
**Pull request by IainTheTechWriter**
Test from https://test.pypi.org/project/python-bitvavo-api/0.0.8/ please.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 .DS_Store
-setup.py
 dist
-python_bitvavo_api.egg-info
\ No newline at end of file
+python_bitvavo_api.egg-info
```
Comment (ksm2) [inline .gitignore:3]: 🥚 ℹ️ ? :-D

File: LICENSE.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ISC License
 
-Copyright (c) 2018, Bitvavo
+Copyright (c) 2023, Bitvavo B.V
```
Comment (ksm2) [inline LICENSE.txt:3]: I think we need an extra dot there:
```suggestion
Copyright (c) 2023, Bitvavo B.V.
```

File: .gitignore [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
+.idea
```
Comment (ksm2) [inline .gitignore:4]: (Missing trailing new line)

File: README.md [hunk 4]
```diff
@@ -51,12 +46,16 @@ Want to quickly make a trading app? Here you go:
...
+    If you installed from `test.pypi.com`, update the requests library: `pip install --upgrade  requests`.   
+
+
 1. **Create a simple Bitvavo implementation**
 
     Add the following code to a new file in your app:
 
     ```python
-    from python_bitvavo_api.bitvavo import Bitvavo
+    from python_bitvavo_api.bitvavo import Bitvavo
```
Comment (ksm2) [inline README.md:58]: There's some extra space there:
```suggestion
    from python_bitvavo_api.bitvavo import Bitvavo
```

File: README.md [hunk 5]
```diff
@@ -79,7 +78,8 @@ Want to quickly make a trading app? Here you go:
     
         # Handle errors.
         def error_callback(self, error):
-            print("Errors:", json.dumps(error, indent=2))
+            print("Add your error message.")
+            #print("Errors:", json.dumps(error, indent=2))
```
Comment (ksm2) [inline README.md:82]: I think I preferred the way it was before, the comment in line 79 is explanatory enough in my view.

File: python_bitvavo_api/__init__.py [hunk 6]
```diff
@@ -0,0 +1 @@
+name = "pythonBitvavoApi"
```
Comment (ksm2) [inline python_bitvavo_api/__init__.py:1]: Missing trailing new line

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+        'requests==2.31.0',
+        'setuptools'
+    ],
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: ISC License (ISCL)",
+        "Operating System :: OS Independent",
+    ],
+    python_requires=">=3.6",
+)
```
Comment (ksm2) [inline setup.py:29]: Missing trailing new line

File: setup.py [hunk 8]
```diff
@@ -0,0 +1,29 @@
+from setuptools import setup, find_packages
+
+# read the contents of your README file
+from os import path
```
Comment (ksm2) [inline setup.py:4]: Please keep imports together on the top, separated by a blank line from the rest:
```suggestion
from setuptools import setup, find_packages
from os import path

# read the contents of your README file
```

File: .gitignore (hunk 1 above)
Comment (IainTheTechWriter) [reply .gitignore:3]: :-)

File: python_bitvavo_api/__init__.py (hunk 6 above)
Comment (IainTheTechWriter) [reply python_bitvavo_api/__init__.py]: Fixed

File: setup.py (hunk 7 above)
Comment (IainTheTechWriter) [reply setup.py]: Fixed

File: .gitignore (hunk 3 above)
Comment (IainTheTechWriter) [reply .gitignore]: Fixed

**Review by ksm2 — APPROVED** [review]

**Pull request merged**